e2etests/stable_test.go

```diff
@@ -448,6 +448,7 @@ eee.shape: document
 eee <- aaa.ccc
 (eee <- aaa.ccc)[0]: '222'
 `,
+			dagreFeatureError: `Connection "(aaa.ccc -- aaa)[0]" goes from a container to a descendant, but layout engine "dagre" does not support this.`,
```

Comment:
Wonder if we should use elk here then instead to test these?
edit: nvm we already test with both, nice!